lib/cli/config.js

```diff
@@ -92,7 +92,7 @@ exports.loadConfig = filepath => {
  * @returns {string|null} Filepath to config, if found
  */
 exports.findConfig = (cwd = utils.cwd()) => {
-  const filepath = findUp.sync(exports.CONFIG_FILES, {cwd});
+  const filepath = find.any(exports.CONFIG_FILES, {cwd});
```

Comment:
In one call, `findUp.sync` is replaced with `find.any`, in another, it's replaced with `find.up`. Is that intentional?

Reply:
Ref https://github.com/lukeed/empathic/blob/main/src/find.ts, they look different, can we stick with `find.up` for now?

Reply:
It is intentional, `up` searches for a single path while `any` searches for multiple:
`find.up(path: string)` vs. `find.any(paths: string[])`